cityofaustin/atd-vz-data
# Extend the local stack script to pg_dump and restore the local DB

## Associated issues

This splits out the local tooling updates from https://github.com/cityofaustin/atd-vz-data/pull/1274 so that we can get this merged into master sooner.

## Testing
**URL to test:** 
Local

**Steps to test:**
1. Start up the local stack by activating the virtual env and then running `./vision-zero replicate-db`
2. You should now have a new snapshot in `/atd-vzd/snapshots/` with today's date
3. Now, run `./vision-zero dump-local-db` to pg_dump the local DB into `/atd-vzd/dumps/`
4. You should now have a new data dump
5. Now you can move that data dump file into the snapshots folder and run `./vision-zero replicate-db -f <your data dump filename>.sql` to restore into the local DB

---
#### Ship list
